ui/admin/app/templates/scopes/scope/credential-stores.hbs

```diff
@@ -0,0 +1,50 @@
+<Rose::Layout::Page as |page|>
+  {{! <page.breadcrumbs>
```

Comment:
Commented till I make them work properly (actually it does not show the proper breadcrumb)